### ISSUE TITLE
VictoriaLogs Web UI: Log query input field requires two submissions for correct query to be sent

### ISSUE THREAD

**Issue by Fusl** [VictoriaMetrics/VictoriaMetrics#7296]
### Describe the bug

After upgrading from VictoriaLogs v0.35.0 to v0.37.0, an issue occurs where submitting a modified log query sends the previously executed query instead of the updated one. The correct query is only sent after submitting it twice.

For example, if the initial query is `*` and you modify it to something else, the first submission will send `*` again, and only after a second submission does the updated query get sent to the API.

### To Reproduce

1. Open up VictoriaLogs VMUI
2. Open up the Network tab in the browser's DevTools and wait for the default query of `*` to finish loading its results
3. Change the query to something else and submit the query
4. In the Network console, observe VMUI sending a query string of `*`
5. Submit the query again
6. Observe VMUI sending the correct query now
7. Change the query to something else again and submit the query
8. Observe VMUI sending a query with the previously submitted query string

### Version

victoria-logs-20241018-003518-tags-v0.37.0-victorialogs-0-g14e33d93ef

**Comment by tiny-pangolin** [comment]
I think was introduced in version 0.36. I have noticed this happens when submitting the query using ctrl  + enter or by pressing the run query button in the UI

**Comment by Fusl** [comment]
This is probably a duplicate of https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7279

**Comment by valyala** [comment]
@Fusl , thanks for pointing to duplicate issue! Closing this issue as duplicate of https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7279 then.
